assets/js/order-script.js

```diff
@@ -1,6 +1,6 @@
 jQuery( function( $ ) {
 
-	$( document.body ).on( 'click', '#doaction, #doaction2', function( e ) {
+	$( '.post-type-shop_order .tablenav' ).on( 'click', '#doaction, #doaction2', function( e ) {
```

Comment:
@alexmigf Since that mentions post types, I'm guessing that's not compatible with HPOS?

Reply:
Actually it is compatible with HPOS, I have checked that before and tested, but since you mentioned it, I agree that we shouldn't rely on something that could be potentially deleted in the future. Fixed.